about.html

```diff
@@ -201,46 +228,59 @@ <h2>Company Overview</h2>
         <h2>Mission and Vision</h2>
         <div class="section-img">
           <img
-            src="https://imgs.search.brave.com/2CAFCbM6U5I6Q03dK-bC8qqqyb_mjrDxo4YGu4-zCnk/rs:fit:500:0:0/g:ce/aHR0cHM6Ly9pbWFn/ZXMuZnJlZWltYWdl/cy5jb20vaW1hZ2Vz/L2xhcmdlLXByZXZp/ZXdzLzMxOC9pbi10/aGUtY2FyLTEtMTQ5/NTA3MC5qcGc_Zm10"
-            alt="Mission and Vision Image"
+            src="https://imgs.search.brave.com/q0e8cW8B-XzUbrgZx4qP0ObqkV3g5cPE82Dptj32vVY/rs:fit:860:0:0/g:ce/aHR0cHM6Ly9pbWcu/ZnJlZXBpay5jb20v/ZnJlZS1waG90by9w/aG90by1jb250ZW50/LzIwMTkvMDMvbXNo/aXN0LzE1NTM5MDA2/bmF0dXJlcmVpbC9z/bGFkZXJzL3ZhbC9j/YWxlbmRhci5qcGc"

```

Comment:
Could you download the image and import from local url ?
This image is not showing up. 

<img width="1141" alt="Screenshot 2024-10-07 at 00 39 26" src="https://github.com/user-attachments/assets/8525d5ae-af21-4da1-ba92-256971039831">